kiwi/schema/kiwi.rng

```diff
@@ -1521,11 +1521,12 @@ definition can be composed by other existing profiles.</a:documentation>
     <define name="k.repository.alias.attribute">
       <attribute name="alias">
         <a:documentation>Alias name to be used for this repository. This is an
-optional free form text. If not set the source attribute
+optional POSIX form text. If not set the source attribute
```

Comment:
This is still free-form text, it's just restricted on what characters are allowed in the field.

Reply:
ok, done. Fixed the schema docstring here and also updated the online documentation which I had forgotten